certificate-renewal/REPWRITERSPECS/BANNO.CDRENEW.V1.POW

```diff
@@ -1283,6 +1323,18 @@ PROCEDURE ERRORHANDLER
  NEWLINE
  PRINT INDENT(1)+Q+ERRORMSG+Q+": "+Q+ERRORDETAIL(ERRORDETAILOFFSET)+Q
 
+
+
+ IF ERRORCODE=503 THEN
+  DO
+   PRINT ","
+   NEWLINE
+   CALL PRINTCURRENTSTATE
+   NEWLINE
+   PRINT "}"
```

Comment:
The code looks good!

Suggestion:
Not necessary, but you could make this PRINT INDENT(2)+"}", since opening is PRINT INDENT(2)+Q+"currentState"+Q+": {".

Reply:
Changed